php/libraries/Candidate.class.inc

```diff
@@ -596,7 +596,7 @@ class Candidate
      *               or NULL if none exist
      * @throws DatabaseException
      */
-    public function getValidSubprojects(): array
+    public function getValidSubprojects(): ?array
```

Comment:
Since this is the major branch why not just update the code to take out the null case and return an empty array instead?

Reply:
Sure I can do that.